### PR TITLE
Add <protocols> to `ipfs id -h` options

### PR DESCRIPTION
**Pull request by Winterhuman**
Adds `<protocols>` to the `ipfs id  --help` text.

### CODE REVIEW

**Comment by Winterhuman** [comment]
@Jorropo Pinging for review

**Comment by Winterhuman** [comment]
@Jorropo Pinging for review

**Comment by Winterhuman** [comment]
@Jorropo Pinging for review

File: core/commands/id.go [hunk 1]
```diff
@@ -52,6 +52,7 @@ If no peer is specified, prints out information for local peers.
 <pver>: Protocol version.
 <pubkey>: Public key.
 <addrs>: Addresses (newline delimited).
+<protocols>: Protocols (newline delimited).
```
Comment (Jorropo) [inline core/commands/id.go:55]:
```suggestion
<protocols>: Libp2p Protocols Registered (newline delimited).
```

**Review by Jorropo — APPROVED** [review]
LGTM thx :tada:

**Pull request merged**